app/src/request-templates.ts

```diff
@@ -50,41 +64,25 @@ export const proofRequestTemplates: Array<ProofRequestTemplate> = [
   {
     id: 'BC:5:19+AndFullName:0.0.1:indy',
     name: '19+ and Full name',
-    description: 'Verify if a person is 19 years end up and full name.',
+    description: 'Verify if a person is 19 years and up and full name.',
     version: '0.0.1',
     payload: {
       type: ProofRequestType.AnonCreds,
       data: [
         {
-          schema: 'XUxBrVSALWHLeycAUhrNr9:2:Person:1.0',
+          schema: personSchema,
           requestedAttributes: [
             {
               names: ['given_names', 'family_name'],
-              restrictions: [
-                // IDIM Person credential
-                { schema_id: 'XpgeQa93eZvGSZBZef3PHn:2:Person:1.0', issuer_did: '7xjfawcnyTUcduWVysLww5' }, // SIT
-                { schema_id: 'KCxVC8GkKywjhWJnUfCmkW:2:Person:1.0', issuer_did: 'KCxVC8GkKywjhWJnUfCmkW' }, // QA
-                { schema_id: 'RGjWbW1eycP7FrMf4QJvX8:2:Person:1.0', issuer_did: 'RGjWbW1eycP7FrMf4QJvX8' }, // Prod
-                // BC Wallet Showcase
-                { schema_id: 'XUxBrVSALWHLeycAUhrNr9:2:Person:1.0', issuer_did: 'XUxBrVSALWHLeycAUhrNr9' }, // Prod
-                { schema_id: '2K2h7kf8VGTLtfoxJgWazf:2:Person:1.1', issuer_did: '2K2h7kf8VGTLtfoxJgWazf' }, // Dev & Test
-              ],
+              restrictions: personRestrictions,
             },
           ],
           requestedPredicates: [
             {
               name: 'birthdate_dateint',
-              predicateType: '>=',
+              predicateType: '<=',
```

Comment:
Are we sure this is a change we want? It seemed to be working before

Reply:
It only works on the same day the credential was issued. If the showcase person credential was issued on the previous day (July 9th) and today is July 10th then the request will fail because July 9th is less than July 10th. The reason it was set to ">=" in the first place was because when the templates were originally created, they used exact age instead of date of birth. ie: "date of birth >= 19" instead of "date of birth >= current date - 19 years" which was a bug because any date_int number would be greater that 19. When I fixed the bug I forgot to flip the direction of the ">=" sign.